src/libraries/System.Private.CoreLib/src/System/IntPtr.cs

```diff
@@ -207,13 +207,14 @@ public unsafe int CompareTo(object? value)
         public unsafe string ToString(IFormatProvider? provider) => ((nint_t)_value).ToString(provider);
         public unsafe string ToString(string? format, IFormatProvider? provider) => ((nint_t)_value).ToString(format, provider);
 
-        unsafe bool ISpanFormattable.TryFormat(Span<char> destination, out int charsWritten, ReadOnlySpan<char> format, IFormatProvider? provider) =>
+        public unsafe bool TryFormat(Span<char> destination, out int charsWritten, ReadOnlySpan<char> format = default, IFormatProvider? provider = null) =>
```

Comment:
Preexisting your change, but it's not obvious to me staring at this now what here requires this be `unsafe`?

Reply:
If I remove `unsafe`, it would become not compilable:
![image](https://user-images.githubusercontent.com/14993992/106197929-40ab2200-61ee-11eb-8f3a-40e727bc128e.png)

Reply:
Ah, `_value` is a pointer.  Ok.